tests/MongoDB.Driver.Tests/Specifications/read-write-concern/OperationTestRunner.cs

```diff
@@ -0,0 +1,108 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
+using MongoDB.Driver.Tests.Specifications.Runner;
+using Xunit;
+
+namespace MongoDB.Driver.Tests.Specifications.read_write_concern
+{
+    public class OperationTestRunner : MongoClientJsonDrivenTestRunnerBase
+    {
+        private HashSet<string> _defaultCommandsToNotCapture;
```

Comment:
Not needed. Instead do this in a constructor:

        public OperationTestRunner()
        {
            DefaultCommandsToNotCapture.Add("find");
        }

Reply:
Good idea. Done